Assets/Scripts/Puzzle/Lever.cs

```diff
@@ -4,14 +4,17 @@
 
 public class Lever : SignalEmitter {
 	[SerializeField] private LeverType leverType = LeverType.Toggle;
-	[SerializeField] float timerEnd = 5f;
-	[SerializeField] private AudioSource audioSource;
+	[SerializeField] float timerEnd = 5f, fastTimeAudioStart;
```

Comment:
`fastTimeAudioStart` should not be a serialized field as you're overwriting it in Start.